README.rst

```diff
@@ -87,6 +87,18 @@ conftest.py file:
               extra.append(pytest_html.extras.html('<div>Additional HTML</div>'))
           report.extra = extra
 
+Creating a single file to the project
+-------------------------------------
+
+In order to respect the `Content Security Policy (CSP) <https://developer.mozilla.org/docs/Web/Security/CSP>`_, the CSS file is created apart by default. But if it is necessary to you create a single file to the whole report, you may run the file by the following way:
```

Comment:
We could give an example of where a self-contained report might be useful. Something like "You can alternatively create a self-contained report, which can be more convenient when sharing your results. This can be done in the following way:".